client/state/selectors/get-email-forwarding-type.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Retrieve the type of the email forwards
+ *
+ * @param  {Object} state    Global state tree
+ * @param  {String} domainName domainName to request email forwards for
+ * @return {Object}          EmailForwards list
```

Comment:
Is this function really going to return an object?

Reply:
Nope, it wouldn't be one of my PRs without some lazy copy and pasting 😁

Reply:
🤣